server.js

```diff
@@ -155,6 +162,17 @@ var initConnection = (ws) => {
     write(ws, queryChainLengthMsg());
 };
 
+var pullConfirmedTransactions = () => {
+    const confirmedTransactions = [];
+    burgerNode.chain.blocks.forEach(block => {
+        block.transactions.forEach(transaction =>{
+            confirmedTransactions.push(transaction);
```

Comment:
We could do Array.concat() here instead of doing a second loop. Saves some time for the server to loop through the entries.

```javascript
let x = [1,2]
let y = [3,4]
x = x.concat(y) // [1,2,3,4]

Reply:
can do